src/Form.php

```diff
@@ -21,6 +22,33 @@ public function __construct()
         $this->attributesLabels = $this->attributeLabels();
     }
 
+    /**
+     * Returns a value indicating whether the attribute is required.
+     *
+     * This is determined by checking if the attribute is associated with a {@see Yiisoft\Validator\Rule\Required}
+     * validation rule.
+     *
+     * Note that when the validator has a conditional validation applied using
```

Comment:
Isn't that OK? i.e. required attribute is usually used in the form HTML that is generated after load() is called.

Can't we actually evaluate "when" condition?

Reply:
Done.